tests/testthat/_snaps/tmb-methods.md

```diff
@@ -748,6 +748,33 @@
     $neg_log_lik
     [1] 1821.197
     
+    $jac_list
```

Comment:
I would maybe not save the whole matrix list etc.... otherwise we are going to get lots of failures on other servers etc.

Reply:
e.g. just expect a list with certain names (`expect_list`, `expect_named`) is probably sufficient

Reply:
good tip, I had already noticed this when I had been adding the report(theta) function to the list. the environment is updated every run of the snapshot test and it fails!
To be clear, we leave $jac_list in the components, but only test the length and names of components(object)?